<launch>
	<remap from="/camera/rgb/image_rect_color" to= "/camera/rgb/image_raw"/>

	<include file="$(find yolo2)/launch/yolo.launch" />
	

</launch>